core/tauri-utils/src/acl/resolved.rs

```diff
@@ -214,45 +182,17 @@ impl Resolved {
       .collect();
 
     let resolved = Self {
-      allowed_commands: allowed_commands
-        .into_iter()
-        .map(|(key, cmd)| {
-          Ok((
-            key,
-            ResolvedCommand {
-              #[cfg(debug_assertions)]
-              referenced_by: cmd.referenced_by,
-              windows: parse_glob_patterns(cmd.windows)?,
-              webviews: parse_glob_patterns(cmd.webviews)?,
-              scope: cmd.resolved_scope_key,
-            },
-          ))
-        })
-        .collect::<Result<_, Error>>()?,
-      denied_commands: denied_commands
-        .into_iter()
-        .map(|(key, cmd)| {
-          Ok((
-            key,
-            ResolvedCommand {
-              #[cfg(debug_assertions)]
-              referenced_by: cmd.referenced_by,
-              windows: parse_glob_patterns(cmd.windows)?,
-              webviews: parse_glob_patterns(cmd.webviews)?,
-              scope: cmd.resolved_scope_key,
-            },
-          ))
-        })
-        .collect::<Result<_, Error>>()?,
-      command_scope: resolved_scopes,
+      allowed_commands,
+      denied_commands,
+      command_scope,
       global_scope,
     };
 
     Ok(resolved)
   }
 }
 
-fn parse_glob_patterns(raw: HashSet<String>) -> Result<Vec<glob::Pattern>, Error> {
+fn parse_glob_patterns(raw: Vec<String>) -> Result<Vec<glob::Pattern>, Error> {
   let mut raw = raw.into_iter().collect::<Vec<_>>();
   raw.sort();
```

Comment:
Unnecessary iterator
```suggestion
fn parse_glob_patterns(mut raw: Vec<String>) -> Result<Vec<glob::Pattern>, Error> {
  raw.sort();
```